library/ix-dev/community/redis/templates/_redis.tpl

```diff
@@ -0,0 +1,101 @@
+{{- define "redis.workload" -}}
+workload:
+  redis:
+    enabled: true
+    primary: true
+    type: Deployment
+    podSpec:
+      hostNetwork: {{ .Values.redisNetwork.hostNetwork }}
+      securityContext:
+        fsGroup: 1001
+      containers:
+        redis:
+          enabled: true
+          primary: true
+          imageSelector: image
+          securityContext:
+            runAsUser: 1001
+            runAsGroup: 0
+            runAsNonRoot: false
+            readOnlyRootFilesystem: false
+          envFrom:
+            - configMapRef:
+                name: config
+          {{ with .Values.redisConfig.additionalEnvs }}
+          envList:
+            {{ range $env := . }}
+            - name: {{ $env.name }}
+              value: {{ $env.value }}
+            {{ end }}
+          {{ end }}
+          probes:
+            {{ $args := printf "-p %v" .Values.redisNetwork.redisPort }}
+            {{ if not .Values.redisConfig.allowEmptyPassword }}
+              {{ $args = printf "%v -a %v" $args .Values.redisConfig.password }}
+            {{ end }}
+            liveness:
+              enabled: true
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  redis-cli {{ $args }} ping | grep -q PONG
+            readiness:
+              enabled: true
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  redis-cli {{ $args }} ping | grep -q PONG
+            startup:
+              enabled: true
+              type: exec
+              command:
+                - /bin/sh
+                - -c
+                - |
+                  redis-cli {{ $args }} ping | grep -q PONG
+      initContainers:
+      {{- include "ix.v1.common.app.permissions" (dict "containerName" "01-permissions"
+                                                        "UID" 1001
```

Comment:
Fixed uid/gid?

Reply:
Yep, container is not playing nice with other uid/his